### PR TITLE
support private registry authenticate

### PR DESCRIPTION
**Pull request by beyondblog**
support private registry authenticate & images

code happy !
: ）

### CODE REVIEW

**Comment by jvanz** [comment]
Cool! +1

**Comment by ehazlett** [comment]
Shipyard v3 has been released and is a major refactor. Could you please do a fresh checkout and rebase?  Thanks!

**Comment by beyondblog** [comment]
Roger

File: controller/static/app/engines/add.html [hunk 1]
```diff
@@ -52,6 +52,32 @@
...
+                    </div>
+                </div>
+                <div class="field">
+                    <label>RegistryPassword</label>
+                    <div class="ui left labeled input">
+                        <input type="text" name="password" ng-model="registry_password">
+                    </div>
+                </div>
+                <div class="field">
+                    <label>RegistryEmail</label>
```
Comment (tombee) [inline controller/static/app/engines/add.html]: Could we make these could we add spaces between the words in the labels?

**Comment by blackshadowwalker** [comment]
:+1:

**Comment by ehazlett** [comment]
Closing as this has been inactive.  Please open a new PR if you want to add this.  Thanks!

**Pull request closed without merging**